packit_service/worker/handlers/fedmsg_handlers.py

```diff
@@ -28,6 +28,7 @@
 from datetime import datetime
 from typing import Type, Optional
 
+from ansible.modules.source_control.gitlab_project import GitLabProject
```

Comment:
Isn't it supposed to be from `ogr`, same as `GithubProject`?

Reply:
Ah, thanks for catch that!

Reply:
Fixed